string type
string id
---
bool success
string message
